### PR TITLE
remove custom operation plugin check from custom api creation

### PR DESCRIPTION
**Pull request by oxarbitrage**
For https://github.com/bitshares/bitshares-core/issues/2032

### CODE REVIEW

**Comment by pmconrad** [comment]
Actually the check is correct (and should even be added for other APIs that require plugins as well, see some other issue). Note that even without the check the error will occur if a node doesn't enable the API, which would be pointless if the plugin is not active.
I agree with @abitmore that the client must handle the case correctly, i. e. `libraries/wallet` in this case.

**Comment by oxarbitrage** [comment]
ok, trying something else.

**Comment by oxarbitrage** [comment]
New approach at https://github.com/bitshares/bitshares-core/pull/2033/commits/860fe964c7ea81b6aefa96721118a8e988b56913

File: libraries/wallet/wallet_api_impl.cpp [hunk 1]
```diff
@@ -56,9 +56,15 @@ namespace graphene { namespace wallet { namespace detail {
...
-        _remote_hist(rapi->history()),
-        _custom_operations(rapi->custom())
+        _remote_hist(rapi->history())
    {
+      try {
+         _custom_operations = rapi->custom();
+      }
+      catch(const fc::exception& e)
+      {
+         wlog("Custom operations plugin is not loaded.");
```
Comment (pmconrad) [inline libraries/wallet/wallet_api_impl.cpp]: Better: `Custom operations API is not active on server.`
What happens in this case if `get_account_storage` is invoked?

Comment (oxarbitrage) [reply libraries/wallet/wallet_api_impl.cpp]: > Better: Custom operations API is not active on server.
Changed.
>  What happens in this case if get_account_storage is invoked?
Crash, added `try` to call, thanks :)

Both changes at https://github.com/bitshares/bitshares-core/pull/2033/commits/f6738ec1108d36a9700641eb31a4c0274302f00f

Note: No need to add `try` at `account_store_map` as it does not use the API. custom operations can be created without the plugin or the api being enabled, not possible to read without them.

**Pull request merged**